activesupport/lib/active_support/multibyte/unicode.rb

```diff
@@ -265,7 +265,7 @@ def tidy_bytes(string, force = false)
       #   Default is ActiveSupport::Multibyte::Unicode.default_normalization_form.
       def normalize(string, form = nil)
         form ||= @default_normalization_form
-        # See http://www.unicode.org/reports/tr15, Table 1
+        # See https://www.unicode.org/reports/tr15, Table 1
```

Comment:
Did you confirmed the page? https://www.unicode.org/reports/tr15/
Looks like that the original link is the right page.